packages/trees/frontend/sirius-components-trees/src/views/ExplorerView.tsx

```diff
@@ -11,8 +11,94 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 import { WorkbenchViewComponentProps } from '@eclipse-sirius/sirius-components-core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useEffect, useRef, useState } from 'react';
+import { TreeToolBar } from '../toolbar/TreeToolBar';
+import { FilterBar } from '../trees/FilterBar';
+import { FilterBarState } from './ExplorerView.types';
 import { TreeView } from './TreeView';
 
-export const ExplorerView = (props: WorkbenchViewComponentProps) => (
-  <TreeView {...props} treeId="explorer://" showToolBar={true} enableMultiSelection={true} treeOptions={undefined} />
-);
+const useStyles = makeStyles((theme) => ({
+  treeView: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'column',
+  },
```

Comment:
With `flexGrow: 1` you are assuming that this generic component is:

- Used in a parent with a flex layout
- The ExplorerView should alway take as much room as possible in said parent

You can not make either of those decisions since we do not know all the use cases in which this component will be reused. The parent should layout the child, a child should not determine how it is layouted in a parent.

Reply:
done